### ISSUE TITLE
Registered global commands going through new registrations on bot restart

### ISSUE THREAD

**Issue by iagolirapasssos** [Pycord-Development/pycord#425]
### Summary

Global commands are being registered every time the bot is restarted

### Reproduction Steps

I am using '@slash_command(name="userinfo", description="Info about a user")' in all global commands as using '@slash_command()' is generating errors. All commands are being successfully registered after 1 hour as expected, but on restarting the bot all global commands are being registered again. So I'm always waiting 1 hour when restarting the bot.

Note: I am using cogs

### Minimal Reproducible Code

```python
##Simple Setup
import discord
from discord.ext import commands

intents = discord.Intents(voice_states=True, messages=True, guilds=True, members=True, emojis=True, bans=True, dm_messages=True, typing=True, invites=True, reactions=True)
bot = discord.Bot(intents=intents)

bot.run("")

##COgs

from discord.ext import commands
from discord.commands import slash_command

class Example(commands.Cog):
    def __init__(self, bot):
        self.bot = bot
 
    @slash_command()
    async def hi(self, ctx):
        await ctx.respond(f"Hi, this is a global slash command from a cog!")

def setup(bot):
    bot.add_cog(Example(bot))
```

### Expected Results

I would like the registered commands to remain registered and not need to wait more than 1 hour for each reboot.

### Actual Results

Registered global commands are being re-registered on every bot restart.

### Intents

voice_states, messages, guilds, members, emojis, bans, dm_messages, typing, invites, reactions

### System Information

- Python v3.9.7-final
- py-cord v2.0.0-alpha
    - py-cord pkg_resources: v2.0.0a4364+g644054f
- aiohttp v3.7.4.post0
- system info: Linux 5.4.0-89-generic #100~18.04.1-Ubuntu SMP Wed Sep 29 10:59:42 UTC 2021

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

If it's not a bug, please teach me how to do the correct procedure by creating new examples with cogs.

**Comment by BobDotCom** [comment]
Will be fixed by #433

**Comment by BobDotCom** [comment]
Follow the aforementioned issue for more information as this issue is being closed. Let me know if this error still occurs once that is done.
